pkg/apply/applydrivers/apply_drivers_default.go

```diff
@@ -273,3 +268,30 @@ func (c *Applier) deleteCluster() error {
 	logger.Info("succeeded in deleting current cluster")
 	return nil
 }
+
+func (c *Applier) syncWorkdir() {
+	workDir := constants.ClusterDir(c.ClusterDesired.Name)
+	sshCmd, err := ssh.NewExecCmdFromRoles(c.ClusterDesired, v2.MASTER)
```

Comment:
if master number is one, same not sync it ?

Reply:
it will skip when local cp.